### ISSUE TITLE
Feature wish: Facilitate identification of organ permeability (in an easy fashion)

### ISSUE THREAD

**Issue by sfrechen** [Open-Systems-Pharmacology/PK-Sim#972]
Currently, if organ permeability is set to "calculated", this parameter cannot be changed in simulations. Also, it cannot be selected for parameter identification. To do so, one would need to reconfigure all simulations and select an alternative value (which has to be defined in the compound building block). This might be quite time consuming in a larger project. 
I suggest to add an "organ permeability scale factor" that can be assessed and changed (default value would be 1).

**Comment by msevestre** [comment]
@sfrechen Another scale factor eh?

I understand where this requirements come from but I fear that this is going to lead to even more confusion and to me it does not feel quite right.

Where would you define the parameter? 

In a simulation? That means there would be no way for you to bring the optimized value back into the compound building block.

In the compound? Then it would be highly confusing because the calculated value would not be the one used. 

>This might be quite time consuming in a larger project.
If a project has grown that large and there was no need to optimize  permeability, I question the workflow to be honest. Shouldn't permeability identification be one of the earlier thing that is being done, as soon as oral data are being used?

So in short, instead of adding another scale parameter, we should maybe think of changing the workflow in PKSim somehow to make this scenario (optimizing calculated permeability) possible

**Comment by sfrechen** [comment]
The topic here is "Facilitate identification of organ permeability (in an easy fashion)" and not "adding another scaling factor" :-) 
A scaling factor (e.g. as simulation parameter or compound parameter) would be one way. Other ideas are welcome.
So please
